apps/arena/lib/arena/game_launcher.ex

```diff
@@ -41,17 +42,28 @@ defmodule Arena.GameLauncher do
     Process.send_after(self(), :launch_game?, 300)
     diff = System.monotonic_time(:millisecond) - state.batch_start_at
 
-    if length(clients) >= @clients_needed or (diff >= @start_timeout_ms and length(clients) > 0) do
-      send(self(), :start_game)
+    cond do
+      length(clients) >= @clients_needed ->
+        send(self(), :start_game)
+
+      diff >= @start_timeout_ms and length(clients) > 0 ->
+        spawn_bots(@clients_needed - Enum.count(state.clients))
+
+      true ->
+        nil
     end
 
     {:noreply, state}
   end
 
   def handle_info(:start_game, state) do
-    {game_clients, remaining_clients} = Enum.split(state.clients, @clients_needed)
+    {game_clients, remaining_clients} =
+      Enum.split(state.clients, @clients_needed)
```

Comment:
Check your formatter, you didn't change anything here and it still got formatted and even more its not following our rules (line_length 120)

Reply:
Hmmm i think i'm using the same formatter as everybody :c i'll restore this but i may need some help with this